decidim-proposals/app/cells/decidim/proposals/proposal_history_cell.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # This cell is used to render the proposal history panel of a resource
+    # inside a tab of a show view
+    #
+    # The `model` must be a proposal resource to get the history from.
+    #
+    # Example:
+    #
+    #   cell(
+    #     "decidim/proposal_history",
+    #     proposal
+    #   )
+    class ProposalHistoryCell < Decidim::ResourceHistoryCell
+      include Decidim::Proposals::ApplicationHelper
+
+      private
+
+      def linked_resources_items
+        # linked resources generate from this proposal
+        [
+          {
+            resources: @model.linked_resources_from(:proposals, "copied_from_component"),
+            link_name: "copied_from_component",
+            text_key: "decidim/proposals/proposal/import_from_proposal_text",
+            icon_key: "Decidim::Proposals::Proposal"
+          },
+          {
+            resources: @model.linked_resources_from(:proposals, "splitted_from_component"),
+            link_name: "splitted_from_component",
+            text_key: "decidim/proposals/proposal/split_from_proposal_text",
+            icon_key: "Decidim::Proposals::Proposal"
+          },
+          {
+            resources: @model.linked_resources_from(:proposals, "merged_from_component"),
+            link_name: "merged_from_component",
+            text_key: "decidim/proposals/proposal/merge_from_proposal_text",
+            icon_key: "Decidim::Proposals::Proposal"
+          },
+          {
+
+            resources: @model.linked_resources(:projects, "included_proposals"),
+            link_name: "included_proposals",
+            text_key: "decidim/budgets/project/text",
```

Comment:
Just a a curiousity : what happens if the budgets module is not installed?
```suggestion
            text_key: "decidim.budgets.project.text",
```

Reply:
If the Budgets module is not installed, there shouldn’t be any links to projects. In any case, if that happens, it would simply display an empty text.